StRoot/StMuDSTMaker/COMMON/StMuFstRawHit.h

```diff
@@ -0,0 +1,77 @@
+/**************************************************************************
+ *
+ * StMuFstRawHit.h
+ *
+ * Author: tchuang 2022
+ **************************************************************************
+ *
+ * Description: Data class for FST hit in StMuDst
+ *
+ **************************************************************************/
+#ifndef StMuFstRawHit_h
+#define StMuFstRawHit_h
+
+#include <TVector3.h>
```

Comment:
TVector3 is not used in the header file (that I can see...) so this include should be removed.

Reply:
Done.